examples/README.md

```diff
@@ -10,7 +10,8 @@ In each example, there is a module declaration such as:
 variable "metal_auth_token" {}
 
 module "openstack" {
-  source = "metal/openstack/metal"
+  # source = "../.."
+  source = "equinix/openstack/metal"
```

Comment:
This project should get moved to `equinix-labs` and be renamed to `terraform-equinix-metal-openstack`